.github/workflows/create_release.yml

```diff
@@ -106,47 +106,54 @@ jobs:
   build-alpine:
     name: alpine
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        docker_platform: [amd64, arm64]
     steps:
     - uses: actions/setup-python@v1
       with:
         python-version: '3.x'
     - uses: actions/checkout@v1
       with:
         submodules: true
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v3
     - name: start docker
       run: |
-        docker run -w /src -dit --name alpine -v $PWD:/src node:lts-alpine
-        echo 'docker exec alpine "$@";' > ./alpine.sh
-        chmod +x ./alpine.sh
+        docker run -w /src -dit --platform=linux/${{ matrix.docker_platform }} --name alpine-${{ matrix.docker_platform }} -v $PWD:/src node:lts-alpine
+        echo 'docker exec alpine-${{ matrix.docker_platform }} "$@";' > ./alpine-${{ matrix.docker_platform }}.sh
+        chmod +x ./alpine-${{ matrix.docker_platform }}.sh
+
 
     - name: install packages
       run: |
-        ./alpine.sh apk update
-        ./alpine.sh apk add build-base cmake git python3 clang ninja py3-pip
+        ./alpine-${{ matrix.docker_platform }}.sh apk update
+        ./alpine-${{ matrix.docker_platform }}.sh apk add build-base cmake git python3 clang ninja py3-pip
```

Comment:
Does the name of the script need to change?   Can't it just continue to be `alpine.sh` since only one of them will ever exist?